portfolio/src/main/webapp/script.js

```diff
@@ -103,3 +103,34 @@ function deleteComment(comment) {
     getComments()
   );
 }
+
+function checkLogin(){
```

Comment:
nit: I was expecting checkLogin() to literally just check if the user has logged in, but there is extra logic here to change the page content based on login status. Consider renaming or splitting this logic up for clarity.

Reply:
Renamed to: displayLoginStatus